internal/tiltfile/tiltfile_test.go

```diff
@@ -977,18 +977,18 @@ func (f *fixture) assertManifest(name string, opts ...interface{}) model.Manifes
 		case []model.PortForward:
 			assert.Equal(f.t, opt, m.K8sInfo().PortForwards)
 		case dcConfigPathHelper:
-			dcInfo := m.DCInfo()
-			if assert.False(f.t, dcInfo.Empty(), "expected a docker-compose manifest") {
+			if assert.True(f.t, m.IsDC(), "expected a docker-compose manifest") {
```

Comment:
🤔 is there weird merge stuff going on? i think this lives in `tiltfile_docker_compose_test.go` on master...?